src/brain.coffee

```diff
@@ -789,6 +789,9 @@ class Brain
     reply = reply.replace(/\\n/ig, "\n")
     reply = reply.replace(/\\#/ig, "#")
 
+    # Prevent empty redirects
+    reply = reply.replace(/\{@\}/ig, "{@*}")
```

Comment:
With debug mode on, does this cause the redirect to look for a reply to a message containing `*`? It would probably work but is kind of a weird way to do it.

* If UTF-8 mode is on the debug output would likely say `Trying to find a reply for "*"`
* In normal mode the literal `*` would be filtered out and it would be `Trying to find a reply for ""`

In either case the only trigger it could possibly match would be the `*` trigger.

It might be better to instead change the regular expression further down in the code that looks for the `{@}` tags. Possibly like:

```coffeescript
match = reply.match(/\{@([^\}]*?)\}/)
```

This would also minimize there being special case filters happening elsewhere in the function and keep all the {@}-related code in one place.